src/Sfa.Das.Sas.Tools.MetaDataCreationTool/Services/XlsxService.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using OfficeOpenXml;
+using Sfa.Das.Sas.Indexer.ApplicationServices.Shared.Settings;
+using Sfa.Das.Sas.Indexer.Core.AssessmentOrgs.Models;
+using Sfa.Das.Sas.Tools.MetaDataCreationTool.Services.Interfaces;
+
+namespace Sfa.Das.Sas.Tools.MetaDataCreationTool.Services
+{
+    public class XlsxService : IXlsxService
```

Comment:
this isn't a generic service for Xlsx files it's specific to Assessment Orgs, could you rename it please